packages/arb-token-bridge-ui/src/components/common/UnsupportedNetworkContent.tsx

```diff
@@ -0,0 +1,47 @@
+import { useWallet } from '@arbitrum/use-wallet'
+import { Web3Provider } from '@ethersproject/providers'
+import { ChainId, getNetworkName } from '../../util/networks'
+import { supportedNetworks, switchChain } from '../../util/NetworkUtils'
+import { Button } from './Button'
+
+export const UnsupportedNetworkContent = () => {
+  const { provider } = useWallet()
+
+  return (
+    <div className="flex flex-col items-center space-y-8 py-24 px-12">
+      <div className="flex w-full justify-center">
+        <span className="center py-4 text-center text-3xl font-medium text-white">
+          Oops! You’re connected to a wrong network.
+        </span>
+      </div>
+
+      {Object.keys(supportedNetworks).map(chainId => (
```

Comment:
Would be great if we could pass these as props! Better to make it an array of objects.